android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/MeasureExtensions.kt

```diff
@@ -117,5 +117,7 @@ suspend inline fun <reified R : Resource> retrievePreviouslyGeneratedMeasureRepo
       filter(MeasureReport.MEASURE, { value = measureUrl })
       operation = queryOperation
     }
+    .asSequence()
     .filter { it.period.start.formatDate(SDF_YYYY_MM_DD) == startDateFormatted }
+    .toList()
```

Comment:
Generally I do not think we need to use a sequence when we eventually need to convert something back to a collection so this usage is not adding any performance improvement.